internal/services/error_renderer.go

```diff
@@ -28,7 +28,7 @@ func renderError(w http.ResponseWriter, r *http.Request, renderer render.Rendere
 	}
 }
 
-func renderNotFoundOrDAOError(w http.ResponseWriter, r *http.Request, err error, daoError string) {
+func renderNotFoundOrDAOError(w http.ResponseWriter, r *http.Request, err error, daoError string, notFoundError string) {
```

Comment:
This parameter does not appear to be used. I think you can simply add `resource string` parameter and this can be passed in a way that both errors look like:

* NotFoundError: Name="not found", Message="[resource] not found", Error="[resource] not found: %w"
* DAOError: Name="dao error", Message="[resource] DAO error", Error="[resource] DAO error: %w"